web/src/regression/org.test.ts

```
@@ -0,0 +1,286 @@
+            await (await driver.findElementWithText('Save changes')).click()
+            await new Promise(resolve => setTimeout(resolve, 500)) // Wait for save
+            await driver.findElementWithText('Save changes')
+
+            {
+                const quicklinks = await getQuickLinks()
+                if (!quicklinks) {
+                    throw new Error('No quicklinks found')
+                }
+                if (!quicklinks.some(l => l.name === quicklink.name && l.url === quicklink.url)) {
+                    throw new Error(
+                        `Did not find quicklink found ${JSON.stringify(quicklink)} in quicklinks: ${JSON.stringify(
+                            quicklinks
+                        )}`
+                    )
+                }
+            }
+
+            // Remove user from org
+            await (await driver.findElementWithText('Members')).click()
+            // eslint-disable-next-line @typescript-eslint/no-misused-promises
+            driver.page.once('dialog', async dialog => {
+                await dialog.accept()
+            })
+            await (await driver.findElementWithText('Leave organization', { wait: { timeout: 1000 } })).click()
+
+            await driver.page.waitForFunction(() => !document.body.innerText.includes('Leave organization'))
+
+            {
+                const quicklinks = await getQuickLinks()
+                if (quicklinks && quicklinks.some(l => l.name === quicklink.name && l.url === quicklink.url)) {
+                    throw new Error(
+                        `Found quicklink ${JSON.stringify(quicklink)} in quicklinks: ${JSON.stringify(quicklinks)}`
+                    )
+                }
+            }
+        })
+    })
+
+    describe('Organizations API', () => {
+        if (process.env.NODE_TLS_REJECT_UNAUTHORIZED !== '0') {
+            console.error('You must set environment variable NODE_TLS_REJECT_UNAUTHORIZED=0 when running this test.')
```

With the `console.error` in the `describe()` block, is the error message correctly linked to the test description in the output? I would put this check at the beginning of the `test()` block and throw an error instead of logging it, so that the failure is reported as accurately as possible, and none of the test body runs.